README.md

```diff
@@ -1,70 +1,84 @@
 
-Kosmos网络的目标是打通多个k8s集群之间的网络，该模块可以独立部署使用。Kosmos网络使`Pod`可以跨集群访问`Pod`、`Service`，就像它们在同一个集 群那样。目前，该模块主要具备以下功能：
-1. **跨集群PodIP、ServiceIP互访**：基于Linux隧道技术，实现了多个k8s集群的L3网络互通，即用户可以在联邦集群范围内进行`Pod-to-Pod`、`Pod-to-Service`访问。
-2. **多模式支持**：对于添加的集群，可以选择`P2P`或者`Gateway`模式，其中`P2P`模式适用于underlay网络互通情况，具有更短的网络路径和更优的性能。`Gateway`模式更具兼容性，适合混合云、多云场景。
-3. **支持全局IP分配**：Kosmos网络允许在联邦集群中存在两个或多个集群使用相同的`Pod/Service`网段，便于用户对存量集群的管理。Kosmos支持配置`PodCIDR/ServiceCIDR` 与 `GlobalCIDR` 的映射关系，`GlobalIP`全局唯一，对于存在网段冲突的集群服务，可以通过`GlobalIP`互访。
-4. **IPv6/IPv4 双栈支持**
+The goal of Kosmos networking is to establish connectivity between multiple Kubernetes clusters. This module can be deployed and used independently. Kosmos networking enables `Pods` to access `Pods` and `Services` across clusters, as if they were in the same cluster. Currently, this module primarily offers the following functionalities:
+1. **Inter-cluster PodIP and ServiceIP Communication**: Utilizing Linux tunneling technology, this enables L3 network connectivity across multiple Kubernetes clusters. This allows users to conduct `Pod-to-Pod` and `Pod-to-Service` communication within the federated cluster scope.
+2. **Multi-Mode Support**: For added clusters, you can choose between `P2P` or `Gateway` mode. `P2P` mode is suitable for underlay network interoperability, offering shorter network paths and superior performance. `Gateway` mode is more compatible and is suitable for hybrid and multi-cloud scenarios.
+3. **Support for Global IP Allocation**: Kosmos networking allows for the presence of two or more clusters within the federated cluster to use the same `Pod/Service` network segments, making it convenient for users to manage existing clusters. Kosmos supports configuring the mapping relationship between `PodCIDR/ServiceCIDR` and `GlobalCIDR`. `GlobalIP` is globally unique, enabling inter-cluster communication for services with conflicting network segments through `GlobalIP`.
+4. **IPv6/IPv4 Dual-Stack Support**
 
-### 网络架构
+### Network Architecture
 
-Kosmos多集群网络模块目前包含以下几个关键组件：
+The Kosmos ClusterLink module currently includes the following key components:
 <div><img src="docs/images/link-arch.png" style="width:900px;" /></div>
 
-- `Controller-Manager`：用于收集所在集群的网络信息，监听网络设置的变化；
-- `Network-manager`：用于计算各个节点需要的网络配置；
-- `Agent`：是一个`Daemonset`，用于配置主机网络，例如隧道创建、路由、NAT等；
-- `Multi-Cluster-Coredns`: 实现多集群服务发现；
-- `Elector`：负责gateway节点选举；
+- `Controller-Manager`：A component is used to gather network information from the current cluster, and it monitors changes in network settings.
+- `Network-manager`：A component is responsible for calculating the network configurations required for each node.
+- `Agent`：This is a DaemonSet used for configuring the host network, including tasks such as tunnel creation, routing, NAT, and so on.
+- `Multi-Cluster-Coredns`: This component facilitates multi-cluster service discovery.
+- `Elector`：It is responsible for conducting gateway node elections.
 
-### 快速开始
+### Quick Start
```

Comment:
It is recommended that the quick start content be updated based on the official account article, what do you think?
Can be used as a separate chapter to describe quickly starting Kosmos, no longer limited to clusterlink.